.github/workflows/checks.yml

```diff
@@ -35,5 +37,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with: 
+          persist-credentials: false
       - name: Test Coverage
         uses: ArtiomTr/jest-coverage-report-action@v2
+        id: coverage
+        with:
+            # tell to the action to not attach comment.
+            output: report-markdown
+            annotations: none
```

Comment:
Let's try this to see if we can get this working the same across forks and the root repo:

```js
            output: ${{ github.event.pull_request.head.repo.full_name == github.event.repository.name && 'comment' || 'report-markdown' }}
            annotations: ${{ github.event.pull_request.head.repo.full_name == github.event.repository.name && 'coverage' || 'none' }}

```

Reply:
done